### PR TITLE
Add demos for `<amp-fx='fade-in-scroll'>`

### PR DESCRIPTION
**Pull request by nainar**

### CODE REVIEW

**Comment by nainar** [comment]
@sebastianbenz could you take a look?

File: src/20_Components/amp-fx-collection.html [hunk 1]
```diff
@@ -132,5 +132,32 @@ <h1 amp-fx="fade-in">
...
+  <!-- The `fade-in-scroll` effect allows an element to fade in as the user scrolls in the viewport.
+
+    Scroll dependent fade in effect can be added to any element by adding `amp-fx="fade-in-scroll"` attribute to an element.
+    You can control the animation via the following attributes:
+    * `data-margin-start` - This parameter determines when to trigger the timed animation. The value specified in `<percent>` dictates that the animation should be triggered when the element is above the specified percentage of the viewport. The default value is `0%`
+    * `data-margin-end` - This parameter determines when to stop the timed animation. The value specified in `<percent>` dictates that the animation should be fully visible when the element crosses this threshold. The default value is `50%`
+    * `data-repeat` - By default once the element is fully visible the opacity of the element is locked in. If the developer wants to change the opacity even after it the element is fully visible please specify this attribute on the element. 
+  -->
+  <!-- **Example 1: Animation starts when the element is 30% above the viewport** -->
+  <div class="header">
```
Comment (sebastianbenz) [inline src/20_Components/amp-fx-collection.html]: class header is not defined.

File: src/20_Components/amp-fx-collection.html [hunk 2]
```diff
@@ -132,5 +132,32 @@ <h1 amp-fx="fade-in">
...
+    <h1 amp-fx="fade-in-scroll" data-margin-start="30%">
+        <span class="title">Lorem Ipsum</span>
+    </h1>
+    <amp-img width="1600" height="900" layout="responsive" src="https://picsum.photos/1600/900?image=981"></amp-img>
+  </div>
+
+  <!-- **Example 2: Animation stops when the element is 80% above the viewport** -->
+  <div>
```
Comment (sebastianbenz) [inline src/20_Components/amp-fx-collection.html]: Do we need the enclosing diff?

File: src/20_Components/amp-fx-collection.html [hunk 3]
```diff
@@ -132,5 +132,32 @@ <h1 amp-fx="fade-in">
...
+    <amp-img amp-fx="fade-in-scroll" data-margin-end="80%" width="1280" height="873" layout="responsive" src="/img/Border_Collie.jpg" alt="Picture of an elephant"></amp-img>
+  </div>
+  <div class="spacer"></div>
```
Comment (sebastianbenz) [inline src/20_Components/amp-fx-collection.html]: Is this needed?

File: src/20_Components/amp-fx-collection.html [hunk 4]
```diff
@@ -132,5 +132,32 @@ <h1 amp-fx="fade-in">
...
+
+  <!-- **Example 3: Make the animation full dependent on the scroll action** -->
+  <div>
```
Comment (sebastianbenz) [inline src/20_Components/amp-fx-collection.html]: Needed?

**Comment by sebastianbenz** [comment]
it's live: https://ampbyexample.com/components/amp-fx-collection/

But I it doesn't seem to work for me.

**Comment by nainar** [comment]
Umm that's strange. It's working on my local machine. 
Is there a difference between the AMP build being used? 

Also I'll just send out a PR to revert this change until I can figure that out in that case.

**Pull request merged**